utils.py

```diff
@@ -74,12 +75,20 @@ def torchtext_iterators(
 
     # Load pretrained embeddings.
     if fasttext_embeds_path is not None:
+        embedding_vectors = torchtext.vocab.Vectors(name=fasttext_embeds_path)
         trg_field.vocab.load_vectors(
-            vectors=torchtext.vocab.Vectors(name=fasttext_embeds_path)
+            vectors=embedding_vectors
         )
         # TODO Remove words without embedding from the output vocabulary
         trg_field.vocab.vectors[trg_field.vocab.stoi[BOS_TOKEN]] = 1/16
         trg_field.vocab.vectors[(trg_field.vocab.vectors == 0).all(-1)] = trg_field.vocab.vectors.mean(0)
+        if is_vivo:
+            full_vocab = torchtext.data.Field()
+            full_vocab.build_vocab(train.trg, min_freq=min_freq)
+            excluded_words = [word for word in full_vocab.vocab.stoi if word not in trg_field.vocab.stoi]
+            excluded_vectors = torch.stack([embedding_vectors[excluded_word] for excluded_word in excluded_words if excluded_word in embedding_vectors.stoi])
```

Comment:
Could you split long lines (just harder to look at on a smaller screen and github :) )

Reply:
Yeah, will do!